.github/workflows/main-build.yml

```diff
@@ -263,16 +263,22 @@ jobs:
       adot-image-name: ${{ needs.build.outputs.staging-image }}
 
   publish-build-status:
-    needs: [ build, contract-tests ]
-    if: ${{ always() }}
-    uses: ./.github/workflows/publish-status.yml
```

Comment:
The publish-status workflow already publishes some metrics around the main build. I did not look into it much, but want to understand why does it currently not work for us and why do we need this change?

Reply:
The workflow publishes both a success and failure metric. For the purposes of our alarms, we only need a failure metric for main build. The workflow is still used by other e2e tests